build-conflicts.php

```diff
@@ -199,6 +226,24 @@ static function (array $components): array {
         )),
     ));
 
+    $prevComposerJSONFileData = file_get_contents(__DIR__ . '/build/roave-security-advisories/composer.json');
+
+    /** @var array $prevComposerDecodedData
+     */
+    $prevComposerDecodedData  = json_decode($prevComposerJSONFileData, true);
+
+    /** @var array<array-key, string> $oldConflictPackages */
+    $oldConflictPackages      = array_keys($prevComposerDecodedData['conflict']);
+
+    $addedAdvisories = [];
+    foreach ($getAdvisories() as $advisory) {
+        if (in_array($advisory->package->packageName, $oldConflictPackages, true)) {
```

Comment:
Well, this would only list **new packages**, but not new versions of those packages :thinking: 

A bit of a more elaborate intersection mechanism is needed here, perhaps reading the current `composer.json` into a set of exclusion ranges (we already have value objects for that)